.github/workflows/run_periodic_tests.yml

```
@@ -26,27 +26,72 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  run_tests:
+  style:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.12
+
+      - name: Check style
+        run: |
+          python -m pip install pre-commit
+          pre-commit run -a
+
+  run_windows_unit_integration_tests:
+    needs: style
+    env:
+      BUILD_IDAKLU: OFF
```

This should be `ON`?

```suggestion
  run_windows_unit_integration_tests:
    needs: style
    env:
      BUILD_IDAKLU: ON
```

---

Yes, I haven't touched periodic tests like we did tests on push. I will fix it